### ISSUE TITLE
Topic 9: iFrame - Aria-label and ANDI Output are incorrect

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#26]
Location: Repetitive Content > Lesson 3: Consistent Identification > Failing examples > Example 3 - Interactive Sample iFrame
Both the aria-label and ANDI Output have incorrect content for the iFrame. It should read: "Repetitive Content Failing **Example 3** - Interactive Sample"
![Top9_iFrame (Lesson3-Failing Example page)](https://user-images.githubusercontent.com/34749031/58579440-0919f480-8219-11e9-9976-9a2a375fb9de.jpg)

**Comment by KSmithO** [comment]
Please re-revview. Interactive sample was actually under wrong number (should have been 2)

**Comment by HD508Github** [comment]
Re-tested and the interactive sample is now under the correct example (i.e., Example 2).  Issue resolved.
